From ben@decadent.org.uk Fri Jun 25 02:35:01 2010
Return-Path: <devel-bounces@linuxdriverproject.org>
X-Flags: 0000
Delivered-To: GMX delivery to s.l-h@gmx.de
Received: (qmail invoked by alias); 25 Jun 2010 00:36:25 -0000
Received: from driverdev.linuxdriverproject.org (EHLO driverdev.linuxdriverproject.org) [140.211.166.17]
  by mx0.gmx.net (mx037) with SMTP; 25 Jun 2010 02:36:25 +0200
Received: from driverdev.linuxdriverproject.org (localhost [127.0.0.1])
	by driverdev.linuxdriverproject.org (Postfix) with ESMTP id 7412646CC3;
	Thu, 24 Jun 2010 17:40:34 -0700 (PDT)
X-Original-To: devel@driverdev.osuosl.org
Delivered-To: devel@driverdev.osuosl.org
Received: from shadbolt.e.decadent.org.uk (shadbolt.e.decadent.org.uk
	[88.96.1.126])
	by driverdev.linuxdriverproject.org (Postfix) with ESMTP id 3915545690
	for <devel@driverdev.osuosl.org>; Thu, 24 Jun 2010 17:39:40 -0700 (PDT)
Received: from [192.168.4.185] (helo=localhost)
	by shadbolt.decadent.org.uk with esmtps
	(TLS1.0:RSA_AES_256_CBC_SHA1:32) (Exim 4.69)
	(envelope-from <ben@decadent.org.uk>)
	id 1ORwso-0002hE-RW; Fri, 25 Jun 2010 01:35:03 +0100
Received: from ben by localhost with local (Exim 4.72)
	(envelope-from <ben@decadent.org.uk>)
	id 1ORwsn-00021K-PV; Fri, 25 Jun 2010 01:35:01 +0100
From: Ben Hutchings <ben@decadent.org.uk>
To: Greg KH <greg@kroah.com>
In-Reply-To: <20100624222123.GC4838@kroah.com>
References: <1276819235.14011.734.camel@localhost>
	<20100622223438.GA3898@kroah.com> <1277340122.26161.45.camel@localhost>
	<20100624222123.GC4838@kroah.com>
Date: Fri, 25 Jun 2010 01:35:01 +0100
Message-ID: <1277426101.26161.168.camel@localhost>
Mime-Version: 1.0
X-Mailer: Evolution 2.30.1.2 
X-SA-Exim-Connect-IP: 192.168.4.185
X-SA-Exim-Mail-From: ben@decadent.org.uk
X-Spam-Checker-Version: SpamAssassin 3.2.5 (2008-06-10) on
	shadbolt.decadent.org.uk
X-Spam-Status: No, score=-1.4 required=5.0 tests=ALL_TRUSTED
	autolearn=disabled version=3.2.5
Subject: [PATCH 1/2] rtl8192s_usb: Remove duplicate device ID
X-SA-Exim-Version: 4.2.1 (built Wed, 25 Jun 2008 17:14:11 +0000)
X-SA-Exim-Scanned: Yes (on shadbolt.decadent.org.uk)
Cc: devel@driverdev.osuosl.org,
 Florian Schilhabel <florian.c.schilhabel@googlemail.com>,
 Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>,
 Guy Sheffer <guysoft@gmail.com>
X-BeenThere: devel@linuxdriverproject.org
X-Mailman-Version: 2.1.12
Precedence: list
List-Id: Linux Driver Project Developer List <devel.linuxdriverproject.org>
List-Unsubscribe: <http://driverdev.linuxdriverproject.org/mailman/options/devel>,
	<mailto:devel-request@linuxdriverproject.org?subject=unsubscribe>
List-Archive: <http://driverdev.linuxdriverproject.org/pipermail/devel>
List-Post: <mailto:devel@linuxdriverproject.org>
List-Help: <mailto:devel-request@linuxdriverproject.org?subject=help>
List-Subscribe: <http://driverdev.linuxdriverproject.org/mailman/listinfo/devel>,
	<mailto:devel-request@linuxdriverproject.org?subject=subscribe>
Content-Type: text/plain;
  charset="us-ascii"
Content-Transfer-Encoding: 7bit
Sender: devel-bounces@linuxdriverproject.org
Errors-To: devel-bounces@linuxdriverproject.org
X-GMX-Antivirus: 0 (no virus found)
X-GMX-Antispam: 0 (Mail was not recognized as spam);
 Detail=5D7Q89H36p6i75npGen84eVAEFK/syJmiNoEBJhgjYKpglu1TZLLw7xMZnJMXwBFK0zrU
 udEInhYyaWAzwtcf5K2pCdD+gZ2/z4PnBLkwixZI+pVtXqOlCN41sOWgjaVeH7+UhPxHlGxFK/rc
 sw7fg==V1;
X-GMX-UID: PsgadG8VYW0+S0NKeWZph7l0amthcxvV
Status: R
X-Status: N
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 drivers/staging/rtl8192su/r8192U_core.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

--- a/drivers/staging/rtl8192su/r8192U_core.c
+++ b/drivers/staging/rtl8192su/r8192U_core.c
@@ -135,7 +135,6 @@ static const struct usb_device_id rtl819
 	{USB_DEVICE(0x0E66, 0x0016)},
 	{USB_DEVICE(0x0b05, 0x1786)},
 	/* these are not in the official list */
-	{USB_DEVICE(0x050d, 0x815F)}, /* Belkin F5D8053 v6 */
 	{USB_DEVICE(0x0df6, 0x004b)}, /* WL-349 */
 	{}
 };
